opentech/apply/funds/templates/funds/submissions.html

```diff
@@ -16,7 +16,9 @@ <h5>Track and explore recent submissions</h5>
 
 <div class="wrapper wrapper--large wrapper--inner-space-medium">
 
-    {% include "funds/includes/round-block.html" with closed_rounds=closed_rounds open_rounds=open_rounds %}
+    {% if closed_rounds or open_rounds %}
```

Comment:
Should be added to this template: https://github.com/OpenTechFund/opentech.fund/blob/master/opentech/apply/dashboard/templates/dashboard/dashboard.html